main.js

```diff
@@ -0,0 +1,40 @@
+const renderer = Renderer()
+const frogGame = FrogGame()
+const timers = []
+
+$("#main-container").on("click", "#start", function() {
+    $("#start").text("Restart")
+    for (timer of timers){
+        clearTimeout(timer)
+        clearInterval(timer)
+    }
+    $(".fa-frog").remove()
+    $("#lose-msg").remove()
+    renderer.run(frogGame.getLevel())
+    $("#level").text(`Level ${frogGame.getLevel()}`)
+    $("#frog-count").text(`${frogGame.getLevel()} Frogs Left`)
+    let time = frogGame.getLevel() + 1
+    $("#count-down").text(`${time} Seconds Left`)
+    time--
+    timers.push(setInterval(function(){
```

Comment:
you can separate this callback function here into another function outside of this main `startGame` function